_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: default
+title: CITE Participation Guide
+collection: tools
+permalink: /tools/citeguide/
+---
+
+# Community Interoperability Test Environment (CITE) Participation Guidance
+
+Prepared By: The FPKI Technical Working Group (TWG)  <br>
+An FPKI Policy Authority Working Group
+
+Last Update: February 12th, 2019 <br>
+
+## Overview
+
+The Federal Public Key Infrastructure (FPKI) has been the subject of various transitions and evolutions during its years of existence. This includes hardware and software upgrades, configuration and architecture changes, implementation of higher complexity keys and algorithms, and implementation of new application capabilities. The impact on continued interoperability throughout these transitions has highlighted the need for an integrated test environment.
+
+The Community Interoperability Test Environment (CITE) was established as the FPKI integrated test environment. CITE provides the FPKI community with a test environment that mimics the production FPKI hierarchy. It is operated by the Federal PKI Management Authority (FPKIMA) and contains a Test Federal Common Policy and Test Federal Bridge. The FPKIMA operated Test Common and Federal Bridge issue test CA certificates to participating Shared Service Providers, Federal Agency PKI, and Non-Federal Affiliates (referred to as FPKI Partners) to perform interoperability and other forms of testing outlined in this document. CITE has two main purposes:
+
+1. Ensure proper functionality of system changes prior to production deployment, and
+2. Identify and resolve technical issues across FPKI Partners.
+
+The FPKI Community can use CITE to evaluate PKI or application changes in a test environment that mimics the production FPKI hierarchy and test potential interoperability issues before those changes are deployed to the FPKI. <br>
+
+## Document Scope
+
+This document provides guidelines and specifications for an FPKI Partner to participate in CITE. CITE Participants refer to an FPKI Partner establishing a test PKI certified or cross-certified with the Test Common Policy or Test Bridge CA. This document shall not define how CITE Participants conduct testing.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+Modifications to this document and any waivers are controlled by the FPKI Technical Working Group. <br>

```

Comment:
What is a waiver in this context?  is there a compliance mechanism that requires a waiver to be granted for auditing purposes or other?

Reply:
Removed